src/app/pages/LandingPage.svelte

```diff
@@ -1,56 +1,189 @@
 <script lang="ts">
   import Upload from "../components/Upload.svelte";
-  import BCP47Tag from "../components//BCP47Tag.svelte";
+  import BCP47Tag from "../components/BCP47Tag.svelte";
 </script>
 
+<style>
+  :global(body) {
+    display: flex;
+
+    min-height: 100vh;
+    margin: 0;
+    padding: 0;
+
+    flex-flow: column nowrap;
+  }
+
+  :global(:root) {
+    --body-max-width: 960px;
+
+    background-color: var(--white);
+
+    scroll-behavior: smooth;
+  }
+
+  main {
+    flex: 1;
+    --quick-start-max-width: calc(var(--body-max-width) / 2);
+
+    margin-bottom: 2rem;
+  }
+
+  .masthead {
+    --wave-height: 322px;
+    --masthead-content-height: 20rem;
+    --masthead-background-color: #09f;
+
+    width: 100vw;
+    padding-bottom: var(--wave-height);
+
+    color: var(--white);
+    background-position: bottom center;
+    background-repeat: no-repeat;
+    background-image: url(assets/wave.svg),
+      linear-gradient(
+        180deg,
+        var(--masthead-background-color) 0%,
+        var(--masthead-background-color) var(--masthead-content-height),
+        rgba(255, 255, 255, 0) var(--masthead-content-height)
+      );
+  }
+
+  .masthead__content {
+    display: flex;
+    width: 80vw;
+    max-width: var(--body-max-width);
+    margin: auto;
+
+    flex-flow: row nowrap;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .masthead__brand {
+    font-weight: bold;
+    color: var(--black, #111);
+  }
+
+  .masthead__title,
+  .masthead__description {
+    font-weight: 400;
+  }
+
+  .explanation {
+    margin: 2rem;
+
+    text-align: center;
+  }
+
+  .quick-start {
```

Comment:
`margin-top` can be added here to add a little whitespace between the arrow and the quick start section.